### ISSUE TITLE
Detected call of `lr_scheduler.step()` before `optimizer.step()

### ISSUE THREAD

**Issue by mmm898** [luyug/Condenser#14]
Hi, I run run_pre_training.py and have this problem: "UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`". I looked for optimize and scheduler in your repo but didn't find anything. Can you help me to fix this. I'm using transformers 4.3.1

**Comment by luyug** [comment]
This is a typical and benign warning when you run with mixed precision and get a overflow.

**Comment by mmm898** [comment]
Thank you for answering. I will close this issue
